app/controllers/public_controller.rb

```diff
@@ -103,6 +103,18 @@ def contact_us
     @title = t('public.contact_us.title')
   end
 
+  def commission_histories
+    if params[:validator_id]
```

Comment:
```suggestion
   commission_histories = if params[:validator_id]
```
and then you can remove assignments to `commission_histories` inside if-else.

Reply:
I don't think so.. Still need to assign @validator only if if params[:validator_id] is present